shell.nix

```diff
@@ -0,0 +1,48 @@
+{ pkgs ? import <nixpkgs> { } }:
```

Comment:
Why is this in the root? Can it be moved elsewhere, to a dedicated folder only containing the "nix" stuff? `hack/nix`?

Reply:
Because shell-nix searches for default.nix and shell.nix in the current directory. That is the default location for all projects so far I have seen that ship with a nix file.